extension/jaegerremotesampling/extension_test.go

```diff
@@ -137,6 +138,8 @@ func TestRemote(t *testing.T) {
 					assert.Equal(t, []string{string(expectedHeaderValue)}, md.Get(expectedHeaderName))
 				}
 			}
+
+			server.Stop()
```

Comment:
better to defer this from L97, to keep start/stop close together

Reply:
Good call, updated.